app/client/src/components/Community.jsx

```diff
@@ -11,16 +13,19 @@ import {
   MenuList,
   Typography,
 } from "@mui/material";
-import FlagTwoToneIcon from "@mui/icons-material/FlagTwoTone";
+// import FlagTwoToneIcon from "@mui/icons-material/FlagTwoTone";
```

Comment:
if we don't need this, please remove

Reply:
Exactly. I added stuff in to be used later